app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/SelectBasisForClaimController.scala

```diff
@@ -136,5 +136,10 @@ object SelectBasisForClaimController {
       .withoutNorthernIrelandClaimsIfApplies(draftClaim)
 
   def getBasisOfClaimsHints(claim: DraftClaim, journeyBindable: JourneyBindable): BasisOfClaimsHints =
-    BasisOfClaimsHints.of(claim) skip (if (journeyBindable === JourneyBindable.Scheduled) 1 else 0)
+    BasisOfClaimsHints
+      .of(claim) skip (if (
+                         journeyBindable === JourneyBindable.Scheduled || journeyBindable === JourneyBindable.Multiple
+                       ) 2
```

Comment:
using magic numbers is not a good idea - can you create a type to represent what you want to achieve and use that to determine what should or should not be displayed

Reply:
I didn't write this part of the code, only amended it to take in the extra journey.